e2e/support/helpers/e2e-dashboard-visualizer-helpers.ts

```diff
@@ -270,21 +270,30 @@ export function chartLegendItem(name: string) {
   return chartLegend().findByText(name);
 }
 
-export function showDashcardVisualizerModal(index = 0) {
+type ShowDashcardVisualizerModalOptions = {
+  buttonText?: "Edit visualization" | "Visualize another way";
+};
+
+export function showDashcardVisualizerModal(
+  index = 0,
+  options: ShowDashcardVisualizerModalOptions = {},
+) {
+  const { buttonText = "Edit visualization" } = options;
   showDashboardCardActions(index);
 
-  getDashboardCard(index)
-    .findByLabelText("Edit visualization")
-    .click({ force: true });
+  getDashboardCard(index).findByLabelText(buttonText).click({ force: true });
 
   modal().within(() => {
     cy.findByTestId("visualization-canvas-loader").should("not.exist");
     dataImporter().findByTestId("loading-indicator").should("not.exist");
   });
 }
 
-export function showDashcardVisualizerModalSettings(index = 0) {
-  showDashcardVisualizerModal(index);
+export function showDashcardVisualizerModalSettings(
+  index = 0,
+  options: ShowDashcardVisualizerModalOptions = {},
```

Comment:
Having a `isVisualizerCard` flag in options and getting the right label text based on it would've been a bit more convenient